### PR TITLE
Make HTML Template

### PR DESCRIPTION
**Pull request by sudhansubhushanmishra**
Created HTML page for displaying terms in table

### CODE REVIEW

**Comment by avie-dev** [comment]
[MUST] @the-lazy-learner put bin/* in the .gitignore because we won't manage them here. It will be just in your local.

**Comment by sudhansubhushanmishra** [comment]
Done. Sorry for forgetting it earlier

**Comment by avie-dev** [comment]
@the-lazy-learner remove the bin/* from the git repository.
you can git rm them

**Comment by sudhansubhushanmishra** [comment]
Ok, done. Sorry again

File: lib64 [hunk 1]
```diff
@@ -0,0 +1 @@
+lib
```
Comment (avie-dev) [inline lib64]: @the-lazy-learner What is this for?

File: pyvenv.cfg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+home = /bin
```
Comment (avie-dev) [inline pyvenv.cfg]: @the-lazy-learner what is this file for? If this is for your local setting no need to push this. Please git ignore and remove this.

**Review by avie-dev — CHANGES_REQUESTED** [review]
Got a few questions and fixes first.

File: lib64 (hunk 1 above)
Comment (sudhansubhushanmishra) [reply lib64]: Ok removed

File: .gitignore [hunk 3]
```diff
@@ -14,6 +14,7 @@ dist/
 downloads/
 eggs/
 .eggs/
+pyenv.cfg
```
Comment (avie-dev) [inline .gitignore]: great. I can still see the pyvenv.cfg here. Please git remove the file itself too

Comment (sudhansubhushanmishra) [reply .gitignore]: Done

File: lib64 (hunk 1 above)
Comment (avie-dev) [inline lib64]: remove this and move back the requirements.txt

**Review by avie-dev — CHANGES_REQUESTED** [review]
I think it's better to put out everything from parent directory pages.
please refer the repo I sent you.

**Review by avie-dev — APPROVED** [review]
I'll merge this first and fix some things

**Pull request merged**